i18n/spec/utilSpec.js

```diff
@@ -1,16 +0,0 @@
-'use strict';
-
-var util = require('../src/util.js');
-
-describe('findLocaleId', function() {
-  it('should find localeId', function() {
-    expect(util.findLocaleId('', 'num')).toBeUndefined();
-    expect(util.findLocaleId('aa', 'datetime')).toBeUndefined();
-    expect(util.findLocaleId('aa', 'randomType')).toBeUndefined();
-    expect(util.findLocaleId('NumberFormatSymbols_en', 'datetime')).toBeUndefined();
-    expect(util.findLocaleId('DateTimeSymbols_en', 'num')).toBeUndefined();
-
-    expect(util.findLocaleId('DateTimeSymbols_en', 'datetime')).toBe('en');
-    expect(util.findLocaleId('NumberFormatSymbols_en_US', 'num')).toBe('en_US');
-  });
-});
```

Comment:
Maybe these could be moved to `closureI18nExtractorSpec`. It has some tests for `findLocaleId`, but there are some more here.